test/TestSpan.cpp

```diff
@@ -245,6 +245,31 @@ static const char *json_template = R"###({
     "timestamp": %lld
 })###";
 
+static const char *json_v2_template = R"###({
+    "traceId": "%016llx%016llx",
+    "name": "test",
+    "id": "%016llx",
+    "parentId": "%016llx",
+    "kind": "CLIENT",
+    "timestamp": %lld,
+    "annotations": [
```

Comment:
to map span 1 format to span 2 involves a little more logic, particularly to remove redundancy (ex cs is the same as span.kind = client + start timestamp). The following is a little more complex than you need because from your tracer, you will only have one host in a span:

https://github.com/openzipkin/zipkin/blob/master/zipkin/src/main/java/zipkin/internal/Span2Converter.java#L43


If you expose a primary recording api in span2 style, like [brave's](https://github.com/openzipkin/brave/blob/master/brave/src/main/java/brave/Span.java) the translation to span1 model is easier 

https://github.com/openzipkin/zipkin/blob/master/zipkin/src/main/java/zipkin/internal/Span2Converter.java#L269